console/src/components/MapCore/MapCore.js

```diff
@@ -146,6 +146,44 @@ const markerLayer = {
 	}
 };
 
+const carLayerId = 'carPoint';
+const addCarLayer = ({coordinates, iconURL, map, orientation}) => {
+	if (map) {
+		// upload png (svg does not work)
+		map.loadImage(iconURL, (error, icon) => {
+			if (error) throw error;
+
+			const carLayer = {
+				'id': carLayerId,
+				'type': 'symbol',
+				'source': {
+					'type': 'geojson',
+					'data': {
+						'type': 'FeatureCollection',
+						'features': [{
+							'type': 'Feature',
+							'geometry': {
+								'type': 'Point',
+								// location of the car
+								'coordinates': coordinates
+							}
+						}]
+					}
+				},
+				'layout': {
+					'icon-image': 'car',
+					'icon-size': 0.10,
+					// rotation of the car
+					'icon-rotate': orientation || 0
```

Comment:
The orientation comes in from Apollo as radians and mapbox accepts degrees. I wonder whether it's best to convert from radians to degrees in the ServiceLayer's data normalization function or here. I expect that degrees make more sense to people mentally, so it may make sense to just convert it at the ServiceLayer once and never have to deal with it again (since it will then always be in degrees).